stm/coqworkmgrApi.mli

```diff
@@ -12,6 +12,9 @@ type priority = Low | High
 val string_of_priority : priority -> string
 val priority_of_string : string -> priority
 
+(* Default priority *)
+val async_proofs_worker_priority : priority ref
```

Comment:
I think this state is no different from the other flags passed to Coqtop.  I think you should put it in your `Coqargs` record

Reply:
Getting the arguments to `AsyncTaskQueue.spawn` still requires quite a bit of surgery. I'd prefer to address that in a different PR as it is certainly orthogonal to this one, and far from trivial.